tensorflow/contrib/learn/python/learn/estimators/logistic_regressor_test.py

```diff
@@ -4,7 +4,7 @@
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
 #
-#     http://www.apache.org/licenses/LICENSE-2.0
+#     https://www.apache.org/licenses/LICENSE-2.0
```

Comment:
For better or worse, the original link is what is in the Apache license.  can we keep it the way it is to keep the headers identical?